### PR TITLE
Add standalone inference code

### PR DESCRIPTION
**Pull request by pedrohlopes**
# PR type

- new feature

# Description

Adding a simple workflow for standalone pythonic inference: Audio np.array as input and audio np.array as output, with easy to change parameters. Also possibilitates file-to-file workflow. This should allow people to plug RVC inference into their own python codebases without much pain.  
A small addition with no impact to current code as a new VC pipeline syntax is created apart from the original one. Up to date with the current dev branch.

### CODE REVIEW

File: config.py
Comment (Tps-F) [inline config.py:1]: It is not used anywhere.

File: configs/config.py
Comment (Tps-F) [inline configs/config.py:1]: You're destroying the config you use elsewhere.

File: configs/config.py [hunk 1]
```diff
@@ -42,22 +42,25 @@ def wrapper(*args, **kwargs):
 
 @singleton_variable
 class Config:
-    def __init__(self):
-        self.device = "cuda:0"
-        self.is_half = True
+    def __init__(self,device='cuda:0',is_half=True, cli_inference=True):
```
Comment (Tps-F) [inline configs/config.py:45]: Why do you need to set up here?
At least cli_inference should be False

File: infer/modules/vc/standalone_infer.py
Comment (Tps-F) [inline infer/modules/vc/standalone_infer.py:1]: Don't create similar files

**Review by Tps-F — CHANGES_REQUESTED** [review]
Almost looks like infer-cli.py
https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/blob/main/tools/infer_cli.py

File: config.py
Comment (pedrohlopes) [reply config.py:1]: right... was a leftover code from first implementation I did. My mistake.

**Pull request closed without merging**